packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.interface.ts

```diff
@@ -3,7 +3,11 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-export interface FilesManagerPanelProps {
+import { ComponentProps } from "react"
+import FilesManagerSearchInput from "App/files-manager/components/files-manager-search-input/files-manager-search-input"
+
+export interface FilesManagerPanelProps
+  extends ComponentProps<typeof FilesManagerSearchInput> {
```

Comment:
Please do not extend component interfaces!

Reply:
why? 😉

Reply:
Because when you are entering this file you should clearly understand what kind of props component takes. Instead of it, you should follow the nested interfaces which is not very convenient.

Reply:
https://github.com/mudita/mudita-center/pull/920/commits/7f9af7ba23fcf793cb511b0742fd568d5b2b8b35